frontend/packages/dev-console/src/components/pipelines/PipelineParameters.tsx

```diff
@@ -27,40 +35,58 @@ const PipelineParameters: React.FC<FormikProps<FormikValues>> = ({
           addLabel="Add Pipeline Params"
           headers={['Name', 'Description', 'Default Value']}
           emptyValues={{ name: '', description: '', default: '' }}
+          isReadOnly={isReadOnly}
         >
-          <InputField name="name" type={TextInputTypes.text} placeholder="Name" />
-          <InputField name="description" type={TextInputTypes.text} placeholder="Description" />
-          <InputField name="default" type={TextInputTypes.text} placeholder="Default Value" />
+          <InputField
+            name="name"
+            type={TextInputTypes.text}
+            placeholder="Name"
+            isReadOnly={!isReadOnly}
```

Comment:
Super confused!

Reply:
When the user has no edit access then ```isReadOnly``` value is false. That is why I am passing ```!isReadOnly```.

Reply:
I changed the variable name to ```pipelineParameterAccess```.